### PR TITLE
[perfect-scrollbar] Updated to v0.7.1. Added global var and vanilla e…

### PR DESCRIPTION
**Pull request by CarbonAtom**
…vents. Improved jsdoc.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/noraesae/perfect-scrollbar>
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/perfect-scrollbar/index.d.ts
Comment (aicest) [inline types/perfect-scrollbar/index.d.ts]: `keyboard'` => `'keyboard'`

File: types/perfect-scrollbar/index.d.ts
Comment (aicest) [inline types/perfect-scrollbar/index.d.ts]: Consider removing this option entirely? Since it don't work.

**Review by aicest — APPROVED** [review]
LGTM

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR appears ready to merge pending express review by a maintainer.

**Pull request merged**
